core/block.ts

```diff
@@ -1628,8 +1646,8 @@ export class Block implements IASTNodeLocation, IDeletable {
       this.interpolate_(
         json['message' + i],
         json['args' + i] || [],
-        // Backwards compatibility: lastDummyAlign aliases implicitDummyAlign.
-        json['lastDummyAlign' + i] || json['implicitDummyAlign' + i],
+        // Backwards compatibility: lastDummyAlign aliases implicitAlign.
+        json['lastDummyAlign' + i] || json['implicitAlign' + i],
```

Comment:
Can we swap these so that 'implicitAlign' has a higher precedence? I think it's weird that if they both exist 'lastDummyAlign' gets applied to end row inputs instead of 'implicitAlign'. I would rather have people move to the cleaner new property.

Can you also update the block factory to use implicitAlign instead of lastDummyAlign?

Reply:
Done.